guide/foundations/backups.md

```diff
@@ -0,0 +1,265 @@
+
+Adding the pass-phrase will generate a different wallet from the recovery-phrase on its own. It is therefore best to add it when first creating the wallet, and before storing any bitcoin in its addresses.
+
+It's worth noting that most hardware wallets support pass-phrases, but most software wallet applications do not.
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/offline-discoveryprotection.png"
+   retina = "/assets/images/guide/foundations/backups/offline-discoveryprotection@2x.png"
+   alt-text = "Offline backup with discovery protection"
+%}
+
+
+**Do** 
+- Use a secure pass-phrase, it can include numbers, special characters and is case sensitive
+- Have redundant and secure backups of both the recovery-phrase and the pass-phrase, they are equally important for a successful wallet recovery
+- Understand you now have two distinct wallets, (one each from the recovery-phrase with or without the pass-phrase), this can be useful to fool a bad actor, or a risk if next of kin is unaware of the pass-phrase 
+
+**Don’t** 
+- Store the recovery-phrase and pass-phrase together
+- Forget to document your setup for next-of-kin
+
+**Suitable for**
+- Large amounts
+- Situations where the backup can be discovered by bad actors
+
+**Alternative**
+Set up a hardware wallet from the recovery and pass-phrase that uses a PIN to protect usage. The PIN now becomes the protection against theft on discovery and should be saved somewhere safe just like the pass-phrase
+
+<br>
+
+#### Level 4 
+## Multiple backups with discovery protection
+One remaining flaw with the previous offline backup levels is that severe damage to the one location where they are stored, could in the worst case scenario lead to the backup not being readable. This could be a fire, flood, earthquake or similar, all more or less likely depending on where you live.
+
+To protect against such a *catastrophic event* we can create multiple backups, and distribute them to other locations. This could be a second home, other family members or a safe deposit box. 
+
+As the risk of discovery increases with the multiplying backups, best practice at this level would be to maintain the protection against theft on discovery like in level 3 for all the additional backups. This could again be by using a pass-phrase, a duplicate hardware wallet with a PIN, or a mix. 
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/multiple-discoveryprotection.png"
+   retina = "/assets/images/guide/foundations/backups/multiple-discoveryprotection@2x.png"
+   alt-text = "Multiple offline backups with discovery protection"
+%}
+
+**Do** 
+- Document your setup well for next-of-kin now that complexity has increased
+
+**Don’t** 
+- Forget to safely store the pass-phrases or PINs
+
+**Suitable for**
+- Large amounts
+- In locations where catastrophic events are common, or likely to cause severe damage
+
+<br>
+
+#### Level 5
+## Multi-continental backups
```

Comment:
I think this should be highlighted as something that we understand may be inaccessible to the majority of people (unless done online which -- don't think we'd be recommending).

Reply:
Yes this level is a bit far-fetched for most people that need this guide.
We could remove it to be honest.

Reply:
Removed.